server/src/routes/clientRouter.js

```diff
@@ -10,7 +10,8 @@ const clientRoutes = [
   "/podcasts",
   "/genres",
   "podcasts/:id",
```

Comment:
It looks like line 12 got merged, but is currently broken.  It should be `/podcasts/:id`